rts/lra/test/pom.xml

```diff
@@ -407,5 +407,18 @@
         </plugins>
       </build>
     </profile>
+    <profile>
+      <id>jdk17</id>
+      <activation>
+        <jdk>[17,)</jdk>
+      </activation>
+      <properties>
+        <jvm.args.modular>--illegal-access=permit
```

Comment:
@mayankkunwar please can you add a comment indicating why these settings are needed?

Reply:
I will add a comment